packages/reactive_ble_mobile/lib/src/reactive_ble_mobile_platform.dart

```diff
@@ -218,6 +218,16 @@ class ReactiveBleMobilePlatform extends ReactiveBlePlatform {
           )
           .then((data) => _protobufConverter.clearGattCacheResultFrom(data!));
 
+  @override
+  Future<int> readRssi(String deviceId) async => _bleMethodChannel
```

Comment:
Don't you think this should be a stream no a Future?

Reply:
I went with a future to match what the underlying APIs did. But after giving it some thought I do agree that a stream would be more useful in terms actual use cases. 
I'll get that changed.

Reply:
Done

Reply:
Sorry open this discussion as you already changed it once, but is a Stream really the best option here? On both Android and iOS, the API is to read the RSSI once. Why would our library add a periodic Stream on top of that? It is more flexible for the user to have the option to read it once, while they can easily write Stream.periodic themselves if they need to.